spyder/config/base.py

```diff
@@ -536,7 +536,7 @@ def translate_gettext(x):
     MAC_APP_NAME = 'Spyder-Py2.app'
 
 
-def running_in_mac_app():
+def running_in_mac_app(pyexec=None):
     """
     Check if Spyder is running inside an app on macOS.

```

Comment:
Let's add some additional info on the docstring on the pyexec paramater

Reply:
can do